tests/test_default_pipeline_configs.py

```diff
@@ -28,6 +28,7 @@ def _noop_generate(self, samples, **gen_kwargs):
 @patch.object(SamplePopulatorBlock, "generate", _noop_generate)
 @patch.object(SelectorBlock, "generate", _noop_generate)
 @patch("instructlab.sdg.llmblock.server_supports_batched", lambda c, m: True)
+@patch.object(Pipeline, "_drop_duplicates", lambda self, dataset, cols: dataset)
```

Comment:
This test is getting uglier and uglier 🫣

Reply:
yeah, can't say I felt proud about adding this line 😆